runtime/compiler/ilgen/Walker.cpp

```diff
@@ -4481,14 +4481,14 @@ TR_J9ByteCodeIlGenerator::genInvoke(TR::SymbolReference * symRef, TR::Node *indi
             //        lconst 0
             //        lconst 0
             //        computeCC
-            //          luadd               adjunct operator
+            //          ladd               adjunct operator
             //            x
             //            y
             TR::Node* y = pop();
             TR::Node* x = pop();
             TR::Node* zero   = TR::Node::create(TR::lconst, 0, 0);
-            TR::Node* luadd  = TR::Node::create(TR::luadd, 2, x, y);
-            TR::Node* carry = TR::Node::create(TR::computeCC, 1, luadd);
+            TR::Node* ladd  = TR::Node::create(TR::ladd, 2, x, y);
+            TR::Node* carry = TR::Node::create(TR::computeCC, 1, ladd);
```

Comment:
Has verification been done to make sure the evaluators for the ladd, luaddc sequence with the computeCC anchor work correctly with this change?

Reply:
The `luadd` to `ladd` and `lusub` to `lsub` conversions did not work correctly. The change produced a segmentation fault during an internal stress test. Going to investigate on the error in the future.